docs/ai/dotnet-ai-ecosystem.md

```diff
@@ -0,0 +1,40 @@
+| [Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/)   | [OpenAI models](https://platform.openai.com/docs/models/overview)<br/>[Azure OpenAI supported models](https://learn.microsoft.com/azure/ai-services/openai/concepts/models)                   | [Semantic Kernel](https://github.com/microsoft/semantic-kernel) (Microsoft)                                 | [docs](https://learn.microsoft.com/semantic-kernel/)     |
+| [Azure OpenAI SDK](https://www.nuget.org/packages/Azure.AI.OpenAI/)                    | [Azure OpenAI supported models](https://learn.microsoft.com/azure/ai-services/openai/concepts/models)                                     | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft)                                | [docs](https://learn.microsoft.com/azure/ai-services/openai/)                                               |
+
+## Connect your data using vector stores
+
+To increase relevancy and tailor AI applications for your own data, you'll likely need to work with a vector store. Many services provide a native SDK for .NET, which you can use directly. You can also use Semantic Kernel, which provides an extensible component model that allows you to try different vector stores without needing to learn each SDK.
+
+| NuGet Package                                                                          | Supported Vector Store            | Maintainer / Vendor                                                                                         | Useful links |
+|----------------------------------------------------------------------------------------|-----------------------------------|-------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| [Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/)   | [Supported vector stores](https://learn.microsoft.com/semantic-kernel/memories/vector-db#available-connectors-to-vector-databases)        | [Semantic Kernel](https://github.com/microsoft/semantic-kernel) (Microsoft)                                 | [docs](https://learn.microsoft.com/semantic-kernel/memories/vector-db)     |
+| [Azure.Search.Documents](https://www.nuget.org/packages/Azure.Search.Documents/)       | Azure AI Search                   | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft)                                | [docs](https://learn.microsoft.com/dotnet/api/overview/azure/search.documents-readme?view=azure-dotnet)     |
```

Comment:
@gewarren For the dotnet/api/overview link we do want to link to the azure-dotnet view explicitly, to be sure the C# examples are shown on that page. Is there any way to do that besides using the ?view= param? Otherwise, I'll add that back in, but appreciate the other edits!

Reply:
Okay, it seemed to me that was the default view. If you want to specify it in the URL, then you should also add ``&preserve-view=true (see https://review.learn.microsoft.com/en-us/help/platform/validation-ref/preserve-view-not-set?branch=main).